src/resources/Incorporations/communityContributionCompany.ts

```diff
@@ -54,24 +54,23 @@ export const CommunityContributionCompanyResource: IncorporationResourceIF = {
   incorporationAgreement: {
     helpSection: [
       {
-        header: 'How do I create an Incorporation Agreement and Articles?',
+        header: 'What is the sample Incorporation Agreement and BC Community Contribution Company Articles?',
         helpText: [
-          'We recommend seeking legal assistance in creating your Incorporation Agreement and Articles.'
+          `The sample Incorporation Agreement and Company Articles is a template that you can use to create an 
+            incorporation agreement and articles for your company. It uses all the standard provisions by legislation. 
+            There are three types of samples depending on if you’re incorporating a Limited Company, a Benefit Company, 
```

Comment:
Does the apostrophe display correctly? Compare with line 83 below.

Reply:
![image](https://user-images.githubusercontent.com/116035339/205148622-c28e649d-761d-47b1-8cdb-c5226aa7d7a1.png)

Reply:
It looks like the browser is displaying `’` and `'` the same (but not here when I escape them)?

OK, not a problem, but consistency would be ideal.